_sass/_syntax-highlighting.scss

```diff
@@ -2,66 +2,199 @@
  * Syntax highlighting styles
  */
 .highlight {
-    background: #fff;
-    @extend %vertical-rhythm;
+  background: #fff;
+  @extend %vertical-rhythm;
 
-    .c     { color: #998; font-style: italic } // Comment
```

Comment:
It looks like this file is from a library, so I’d say we exempt it from the linter?
Otherwise: Where are these styles used anyway?